internal/admin/server.go

```diff
@@ -6,22 +6,30 @@
 package admin
 
 import (
+	"embed"
 	"net/http"
 	"net/http/pprof"
 	"time"
 
 	"github.com/davecgh/go-spew/spew"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/envoyproxy/gateway/internal/envoygateway/config"
 )
 
-func Init(cfg *config.Server) error {
+//go:embed webui.html logo.svg
+var webuiFS embed.FS
+
+func Init(cfg *config.Server, k8sClient client.Client) error {
 	if cfg.EnvoyGateway.GetEnvoyGatewayAdmin().EnableDumpConfig {
 		spewConfig := spew.NewDefaultConfig()
 		spewConfig.DisableMethods = true
 		spewConfig.Dump(cfg)
 	}
 
+	// Set the k8s client for handlers
```

Comment:
can we rm k8s dependancy
https://github.com/envoyproxy/gateway/pull/6331/files#r2198490409